.github/workflows/050-release-ubuntu-make-build-release.yml

```diff
@@ -65,7 +62,7 @@ jobs:
                 key: release-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
             - name: Make release artifacts
-              run: make release
+              run: make musl build release
```

Comment:
'make musl proto release' should work. Better yet, add the proto step to the makefile like it is for other make commands

Reply:
Ah right, I forgot the proto step is better than the build. Thanks, Ill fix that.